src/commandline_frame.ts

```diff
@@ -279,7 +279,7 @@ function process() {
 
     hide_and_clear()
 
-    const [func, ...args] = command.trim().split(/\s+/)
+    const [func, ...args] = command.trim().split(/ +/)
```

Comment:
I think the "colon ignoring" should go here, really. Colon is a message to the command line that we want the line to be parsed as if it was an ex-command. I don't think it's reasonable to expect aliases to have colons ignored.

Reply:
Prepended colons become part of the excmd language with this PR. Their handling should go in src/parsers/exmode.ts:parser().

In vim prepended colons are ignored in maps, composite statements, etc: `:echo "hi" | :::echo "bye"`

Reply:
It's not really anything to do with aliases, IMO. Aliases should be expanded, then colons should be removed, then other parsing things.